test/integration/backends/ibis/test_end_2_end.py

```diff
@@ -186,21 +190,23 @@ def test_nested_query(clean_cache):
         },
     )
 
-    t = IbisQuery(identifier='my_table', schema=schema)
+    from superduperdb.components.table import Table
+    t = Table(identifier='my_table', schema=schema)
 
     db.add(t)
 
-    q = t.filter(t.age > 10)
+    t = db['my_table']
+    q = t.filter(t.age >= 10)
 
-    expr_, _ = q.compile(db)
+    expr_ = q.compile(db)
 
     if not memory_table:
-        assert 'SELECT t0.id, t0.health, t0.age, t0.image, t0._fold' in str(
-            expr_.compile()
+        assert 'SELECT t0._fold, t0._schema, t0.id, t0.health, t0.age, t0.image' in str(
```

Comment:
Yes I don't think we need to add `_schema` to the table, since it it the same for every line of the 
table.